code/ensemble/utils/use-ensemble-criteria.R

```diff
@@ -0,0 +1,60 @@
+# Load models according to inclusion criteria before ensembling
+# 
+# Steps:
+# Currently, models included based on having: 
+# 1. All quantiles
+# 2. 4 horizons 
+# 3. Not manually specified for exclusion 
+# 4. Not the hub ensemble
+# 
+# Returns a list:
+# - "forecasts" = tibble, all forecasts passing inclusion criteria
+# - "criteria" = tibble, model names and criteria assessment
+# 
+library(dplyr)
+library(tibble)
+
+use_ensemble_criteria <- function(forecasts = 
+                                    covidHubUtils::load_forecasts(source = "local_hub_repo",
+                                                                  hub_repo_path = here(),
+                                                                  hub = "ECDC",
+                                                                  forecast_dates = forecast_date),
```

Comment:
could make forecast_date the first argument to the function here I think? (assuming the second argument has access to the first argument)

Reply:
I'm not sure it does have access, so have defined in both places to be safe.